packages/odyssey/src/scss/abstracts/_tokens.scss

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018-Present Okta, Inc.
```

Comment:
Related to my comment above, how many tokens do we want to doc and should we think more about how to prefix them? Do we want to group by value type? Use case?

spitballing: `$size-text-`, `$size-spacing-`, `$color-focus-`, `$color-type-`, etc

Reply:
My intention here was not to provide finalized naming around tokens, as a result, I would really prefer it if no one used them right now. That said it's looking like there's a strong desire to use this soon as part of the phase 1 work. Let's discuss this and come up with a convention we all feel good about. (I don't disagree with what you're proposing, just want to make sure Joe is looped in)